spec/features/phone_configuration/phone_configuration_spec.rb

```diff
@@ -0,0 +1,147 @@
+require 'rails_helper'
```

Comment:
How you you feel about moving this somewhere like `/users/default_phone_selection_spec.rb` or something like that?